src/RedisInterface/Controllers/actionPlanModelController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Middleware.Common.Models;
+using Middleware.Common.Repositories;
+
+namespace Middleware.RedisInterface.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    public class actionPlanModelController : ControllerBase
```

Comment:
Adrian, we don't put 'Model' in the controller names.  Also I believe that the endpoints for the ActionPlan are located in the `ActionController` :)
The function that you have put in here is already implemented there